README.md

```diff
@@ -1 +1,24 @@
-# Blog_App
\ No newline at end of file
+# README
```

Comment:
- [ ] Kindly update your readme with the template from Microverse as requested earlier